README.md

```diff
@@ -59,8 +59,8 @@ The design philosophy of uVisor is to provide hardware-enforced compartments (sa
 
 Breaking the established flat security model of micro-controllers into compartmentalised building blocks results in high security levels, as the reach of flaws or external attacks can be limited to less sensitive function blocks.
 
-A [basic example of the uVisor](https://github.com/ARMmbed/mbed-os-example-uvisor) features is preventing unauthorised access to flash memory from faulty or compromised code. This not only prevents malware from getting resident on the device, but also enables protection of device secrets like cryptographic keys.
-For interrupt support in uVisor please refer to our [uVisor interrupt example](https://github.com/ARMmbed/mbed-os-example-uvisor-irq). 
+A [basic example of the uVisor](https://github.com/ARMmbed/mbed-os-example-uvisor) demonstrates features to prevent unauthorised access to flash memory from faulty or compromised code and interrupts. This not only prevents malware from getting resident on the device, but also enables protection of device secrets like cryptographic keys.
+For threaded support in uVisor please refer to our [uVisor threading example](https://github.com/ARMmbed/mbed-os-example-uvisor-thread).
```

Comment:
In this line you inverted `threaded` and `threading` wrt to line 19? I prefer the version you wrote in line 19